### ISSUE TITLE
test..com is valid domain

### ISSUE THREAD

**Issue by buzanits** [layershifter/TLDExtract#22]
```
$result = $extract->parse('test..com');
$result->isValidDomain();  // gives true
```
Registrable Domain ist .com in this case... I think this domain name should be invalid.

**Comment by layershifter** [comment]
Thanks for report, valid issue. I will fix it before 1st April.

**Comment by layershifter** [comment]
Thanks for report, fixed in the latest release `1.2.4`. Please check :+1:
